src/utils/param-converter/convert-nested-param.ts

```diff
@@ -16,6 +16,8 @@ const convertNestedParam = (
     const nestedSubProperties = subProperty.subProperties
 
     for (const nestedSubProperty of nestedSubProperties) {
+      if (!value) continue;
```

Comment:
This would simply make it skip all loops, changing the `if` condition to:

```javascript
 if (type === 'mixed' && value) {
```

should make more sense.